ehsm_kms_service/ehsm_kms_server.js

```diff
@@ -352,18 +164,30 @@ app.post('/ehsm', function (req, res) {
     const { cmk_base64, keylen, aad } = PAYLOAD;
     napi_result(ACTION ,res, [cmk_base64, keylen, aad]);
   } else if(ACTION === apis.Sign) {
-    /**
-     * Sign
-     */
-      const { cmk_base64, digest } = PAYLOAD;
-      napi_result(ACTION ,res, [cmk_base64, digest]);
+  /**
+   * Sign
+   */
+    const { cmk_base64, digest } = PAYLOAD;
+    napi_result(ACTION ,res, [cmk_base64, digest]);
   } else if(ACTION === apis.Verify) {
-    /**
-     * Verify
-     */
-      const { cmk_base64, digest, signature_base64 } = PAYLOAD;
-      napi_result(ACTION ,res, [cmk_base64, digest, signature_base64]);
-  }else {
+  /**
+   * Verify
+   */
+    const { cmk_base64, digest, signature_base64 } = PAYLOAD;
+    napi_result(ACTION ,res, [cmk_base64, digest, signature_base64]);
+  } else if(ACTION === apis.AsymmetricEncrypt) {
+  /**
```

Comment:
Will js layer check the input validity, e.g. the maxlength, minLength？

Reply:
It will be processed in the JS layer